src/bundle/Resources/public/js/scripts/helpers/backdrop.helper.js

```diff
@@ -0,0 +1,34 @@
+(function (global, doc, ibexa) {
+    class Backdrop {
+        constructor(config = {}) {
+            this.isTransparent = config.isTransparent ?? false;
+            this.extraClasses = config.extraClasses ?? [];
+            this.backdrop = null;
+
+            this.remove = this.remove.bind(this);
+            this.init = this.init.bind(this);
+        }
+
+        remove() {
+            if (this.backdrop) {
+                this.backdrop.remove();
+            }
+        }
+
+        init() {
+            const classes = {
+                'ibexa-backdrop--transparent': this.isTransparent,
+            };
+            const backdropClasses = Object.keys(classes).filter((property) => classes[property]);
+            const backdropExtraClasses = Array.isArray(this.extraClasses) ? this.extraClasses : [this.extraClasses];
```

Comment:
I'm not sure about having mixed type here, string or array. I would suggest using only array.

Reply:
I like to have flexible code so later we can send one string as extra classs or array but I can change it to array only no problem.